.github/workflows/test-environment.yml

```diff
@@ -137,7 +137,13 @@ jobs:
         id: apply
         if: success()
         run: |
-          terraform apply --auto-approve -var="deployment_name=${{ env.DEPLOYMENT_NAME }}" -var="region=${{ env.AWS_REGION }}"
+          terraform apply --auto-approve \
+            -var="deployment_name=${{ env.DEPLOYMENT_NAME }}" \
+            -var="region=${{ env.AWS_REGION }}" \
+            -var="division=engineering" \
+            -var="org=security" \
+            -var="team=cloud-security-posture" \
+            -var="project=${{ github.actor }}"
```

Comment:
@moukoublen Since all tags have default values, you only need to provide the `project` variable. The other tags will use their default values.